test/fudge/log_test.clj

```diff
@@ -126,33 +134,33 @@
       (let [logger (plain-logger)
             result (with-out-str
                      (log logger :info "foo"))]
-        (is (= "2017-05-12T18:01 [fudge.log-test:128] [info] foo\n" result))))
+        (is (= "2017-05-12T18:01 [fudge.log-test:136] [info] foo\n" result))))
```

Comment:
I did also consider pulling out specifically the line number parsing from the metadata to a function of it's own, then redefining that function for the tests. Problem is you can't redefine the `calling-meta` macro directly, and I wanted to test that it was tied in to the rest correctly